core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.scss

```diff
@@ -68,12 +68,9 @@
         background: $color-palette-gray-200;
     }
 
-    .p-tabview .p-tabview-nav li.p-highlight .p-tabview-nav-link {
-        border-bottom: 4px solid $color-palette-secondary-500;
-    }
-
     .p-tabview .p-tabview-nav li:not(.p-highlight):not(.p-disabled):hover .p-tabview-nav-link {
-        border-bottom: 4px solid $color-palette-secondary-500;
+        border-top: 1px solid #afb3c0;
+        border-bottom: 1px solid #afb3c0;
```

Comment:
We should use variables here and anywhere else we are uding this color.

Reply:
Also please check the height with the one in content types and anything else that looks different here, this tabs should looks the same as the one in content types.

Reply:
I think there are custom tab code here that we don't need anymore.

Reply:
check `_colors.scss` I think this is both `$color-palette-gray` and  `$color-palette-gray-500`